browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchClient.java

```diff
@@ -4,331 +4,92 @@
 import android.content.Intent;
 import android.net.Uri;
 
-import androidx.annotation.Nullable;
+import androidx.annotation.NonNull;
 import androidx.annotation.VisibleForTesting;
-import androidx.fragment.app.Fragment;
+import androidx.browser.customtabs.CustomTabsIntent;
 import androidx.fragment.app.FragmentActivity;
 
 import org.json.JSONObject;
 
-@SuppressWarnings("WeakerAccess")
 public class BrowserSwitchClient {
```

Comment:
We kept all the files in this module under the `com.braintreepayments.browserswitch` package for a clean diff, but this module will also be moved to `com.braintreepayment.api` to take advantage of `package-private` scope.